<!-- ROBOT -->
<launch>

  <arg name="sim" default="true" />

  <push_ros_namespace namespace="$(env ROBOT_NAME)" />

  <set_parameter name="use_sim_time" value="true" if="$(var sim)" />

  <node pkg="rviz2" exec="rviz2"
      args="-d $(find-pkg-share integrated_planner)/launch/robot_launch_gazebo/gazebo_vis.rviz --ros-args --log-level INFO"
      output="screen" respawn="true" respawn_delay="1" />
  
  <!-- Static TFs -->
  <include file="$(find-pkg-share integrated_planner)/launch/robot_launch_gazebo/gz_static_transforms.launch.xml" />

  <!-- Autonomy -->
  <!-- <include file="$(find-pkg-share integrated_planner)/launch/robot_launch_gazebo/gz_autonomy_launch.xml" /> -->

  <!-- Local -->
  <include file="$(find-pkg-share integrated_planner)/launch/robot_launch_gazebo/gz_local_launch.xml" />

  <!-- Global -->
  <node pkg="integrated_planner" exec="cpa_node" name="cpa_node" output="screen" emulate_tty="true">
    <param from="$(find-pkg-share integrated_planner)/config/fov_gazebo_config.yaml" />
  </node>

  <!-- Logging -->
  <!-- <include file="$(find-pkg-share logging_bringup)/launch/logging.launch.xml" /> -->

  <!-- Domain Bridge -->
  <!-- <node pkg="domain_bridge" exec="domain_bridge"
    args="$(find-pkg-share integrated_planner)/launch/robot_launch_gazebo/gz_domain_bridge.yaml"
    output="screen" respawn="true" respawn_delay="1" /> -->

</launch>